src/backend/src/utils/auth.utils.ts

```diff
@@ -36,9 +37,12 @@ export const requireJwtProd = (req: Request, res: Response, next: any) => {
 
     if (!token) return res.status(401).json({ message: 'Authentication Failed: Cookie not found!' });
 
-    jwt.verify(token, TOKEN_SECRET, (err: any, decoded: any) => {
+    jwt.verify(token, TOKEN_SECRET, (err: VerifyErrors | null, decoded: string | JwtPayload | undefined) => {
```

Comment:
I beleive this should be `string | JwtPayload | Jwt` based on the dependency file, is there a specific reason you left it out?

Reply:
Of the four asynchronous verify functions in the index.d.ts file, only the first one takes 3 parameters (on line 215). The other four take in an VerifyOptions as well as the token, key, and VerifyCallback. In the function with 3 parameters, the callback parameter is typed as VerifyCallback<JwtPayload | string>. It was my understanding that the JwtPayload | string becomes the T in the VerifyCallback type, which is why I didn't include the Jwt in the decoded type. That being said, I can add the Jwt type in and check for it the same as the string if you think that would be better, or I can explain the stuff I just explained here in the pr if you think it makes sense.